haiku-apps/bescreencapture/bescreencapture-2.2.1.recipe

```diff
@@ -15,9 +15,10 @@ COPYRIGHT="2014-2016 Stefano Ceccherini"
 LICENSE="BSD (3-clause)
 	MIT"
 REVISION="1"
-SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="779a7ac4e4e7b935f31ff95b47ab1325a81a98d9fe5d76017fb8c4e01b3ed5ac"
-SOURCE_FILENAME="bescreencapture-$portVersion.tar.gz"
+SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v"$portVersion"a.tar.gz"
+CHECKSUM_SHA256="d36f34d0dcf8bb87d58d782e649f29862fbbaed68a26182ccb695b0b0716ec87"
+SOURCE_FILENAME="bescreencapture-"$portVersion"a.tar.gz"
+SOURCE_DIR="bescreencapture-"$portVersion"a"
```

Comment:
Oops, I assumed everyone was OK with keeping «**`2.2.1`**» for the version (instead of «**`2.2.1a`**»), but actually I don't know if all other major contributors also agree with that. (So comments are welcome, as usual!)

Reminder: I'm OK with keeping «**`2.2.1`**» for the version, although I would have personally used «**`2.2.1a`**».

Regarding the  «**`"$portVersion"a`**» vs «**`${portVersion}a`**», however, I think the style of the latter preferable.